vendor/bat-native-ledger/src/bat/ledger/internal/ledger_impl.h

```diff
@@ -129,7 +130,13 @@ class LedgerImpl : public ledger::Ledger,
                         int year,
                         const ledger::BalanceReportInfo& report_info);
 
-  void SaveUnverifiedContribution(ledger::PendingContributionList list);
+  void SaveUnverifiedContribution(
+      ledger::PendingContributionList list,
```

Comment:
Type should be `const ledger::PendingContributionList&` I think.

Reply:
The implementation of this function calls `std::move(list)`, which will not work with `const ledger::PendingContributionList&`. The `std::move` is necessary because `ledger::PendingContributionList` is a `std::vector` of `mojom::StructPtr`s, and so it has no copy constructor.

Reply:
OK, got it.